tests/models/test_partner_api_token.py

```diff
@@ -0,0 +1,25 @@
+from app.models import Partner, PartnerApiToken
+from app.utils import random_string
+
+
+def test_generate_partner_api_token(flask_client):
+    partner = Partner.create(
+        name=random_string(10),
+        contact_email="{s}@{s}.com".format(s=random_string(10)),
+        commit=True,
+    )
+
+    (partner_api_token, token) = PartnerApiToken.generate(partner.id, None)
```

Comment:
Nits: we can remove parenthesis `(` here :)

Reply:
Removed